src/vm/docs/mod.rs

```diff
@@ -611,26 +611,117 @@ control-flow.",
 const EXPECTS_API: SpecialAPI = SpecialAPI {
     input_type: "(optional A) | (response A B), C",
     output_type: "A",
-    signature: "(expects! option-input thrown-value)",
-    description: "The `expects!` function attempts to 'unpack' the first argument: if the argument is
-an option type, and the argument is a `(some ...)` option, `expects!` returns the inner value of the
-option. If the argument is a response type, and the argument is an `(ok ...)` response, `expects!` returns
+    signature: "(unwrap! option-input thrown-value)",
+    description: "The `unwrap!` function attempts to 'unpack' the first argument: if the argument is
+an option type, and the argument is a `(some ...)` option, `unwrap!` returns the inner value of the
+option. If the argument is a response type, and the argument is an `(ok ...)` response, `unwrap!` returns
  the inner value of the `ok`. If the supplied argument is either an `(err ...)` or a `(none)` value,
-`expects!` _returns_ `thrown-value` from the current function and exits the current control-flow.",
-    example: "(expects! (map-get names-map (tuple (name \"blockstack\"))) (err 1)) ;; Returns (tuple (id 1337))",
+`unwrap!` _returns_ `thrown-value` from the current function and exits the current control-flow.",
+    example: "(unwrap! (map-get names-map (tuple (name \"blockstack\"))) (err 1)) ;; Returns (tuple (id 1337))",
+};
+
+const TRY_API: SpecialAPI = SpecialAPI {
+    input_type: "(optional A) | (response A B)",
+    output_type: "A",
+    signature: "(try! option-input)",
+    description: "The `try!` function attempts to 'unpack' the first argument: if the argument is
+an option type, and the argument is a `(some ...)` option, `try!` returns the inner value of the
+option. If the argument is a response type, and the argument is an `(ok ...)` response, `try!` returns
+ the inner value of the `ok`. If the supplied argument is either an `(err ...)` or a `none` value,
+`try!` _returns_ either `none` or the `(err ...)` value from the current function and exits the current control-flow.",
+    example: "(try! (map-get names-map (tuple (name \"blockstack\"))) (err 1)) ;; Returns (tuple (id 1337))
+(define-private (checked-even (x int))
+  (if (eq? (mod x 2) 0) 
+      (ok x)
+      (err 'false)))
+(define-private (double-if-even (x int))
+  (ok (* 2 (try! (checked-even x)))))
+(double-if-even 10) ;; Returns (ok 20)
+(double-if-even 3) ;; Returns (err 'false)
+",
+};
+
+const UNWRAP_API: SpecialAPI = SpecialAPI {
+    input_type: "(optional A) | (response A B)",
+    output_type: "A",
+    signature: "(unwrap option-input)",
+    description: "The `unwrap` function attempts to 'unpack' its argument: if the argument is
```

Comment:
I'm not sure -- I think that `unwrap-or-panic` is more self-explanatory, but it ends up being a little verbose -- I'm not as worried about the proximity in naming as I would be otherwise, since `unwrap` and `unwrap!` have different argument signatures, so any misuse would get picked up by the typechecker.